src/hasher.rs

```diff
@@ -1,37 +1,38 @@
-use std::hash::{Hasher, BuildHasher};
-use std::collections::{HashMap, HashSet};
-use std::mem::MaybeUninit;
+use crate::gxhash::{platform::*, *};
 
 use rand::RngCore;
-
-use crate::gxhash::*;
-use crate::gxhash::platform::*;
+use std::{
+    collections::{HashMap, HashSet},
+    hash::{BuildHasher, Hasher},
+    mem::MaybeUninit,
+};
```

Comment:
I don't think it is any better

Reply:
Yeah, heaps sorry.

This is something I had in a global `rustfmt.toml` that shouldn't have been applied.